Installation pages for specific packages are missing an important step

### What happened?

If one tries to follow the instructions for a package installation given on a particular package page in the Storybook ([Angular](https://e-gov.github.io/cvi/?path=/docs/angular-installation--installation), [styles](https://e-gov.github.io/cvi/?path=/docs/styles-how-to-install--how-to-install), [icons](https://e-gov.github.io/cvi/?path=/docs/icons-how-to-install--installation)), they might stumble upon this error:

```
npm ERR! 404  '@egov/cvi-ng@1.15.2' is not in this registry
```

This is because one important step is missing – configuring the `@egov` registry. 

The step is mentioned in the project README file. 

### Reproduction steps

1. Create a new Angular project
1. Go to a How to install page for a package (eg [Angular](https://e-gov.github.io/cvi/?path=/docs/angular-installation--installation))
2. Follow the instructions given there
4. See the 404 error upon executing `npm install` – the installation of the package will fail


### Screenshot

_No response_

### Package

Not applicable

### Version

Any

### Reproduction URL

_No response_

### Browser

_No response_

### Browser version

_No response_

### Extra details about your environment

_No response_

### Relevant log output

```shell
npm i @egov/cvi-ng
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@egov/cvi-ng - Not found
npm ERR! 404
npm ERR! 404  '@egov/cvi-ng@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/libs/storybook/src/lib/stories/angular.stories.mdx
+++ b/libs/storybook/src/lib/stories/angular.stories.mdx
@@ -31,12 +31,14 @@ import uiPkg from '/libs/ui/package.json'
 
 ## Installation
 
-1. Install the library
+1. Add the `@egov` registry to your npm configuration (see the first step in the [general Installation instructions](https://github.com/e-gov/cvi#%EF%B8%8F-installation-instructions)).
+
+2. Install the library
 ```
 npm i --save @egov/cvi-ng
 ```
 
-2. Import `UiModule` in your `app.module.ts`.
+3. Import `UiModule` in your `app.module.ts`.
 
 ```ts
 import { UiModule } from '@egov/cvi-ng';

--- a/libs/storybook/src/lib/stories/icons/installation.stories.mdx
+++ b/libs/storybook/src/lib/stories/icons/installation.stories.mdx
@@ -14,7 +14,9 @@ import iconsPkg from '/libs/icons/package.json'
   }}
 </Story>
 
-1. Install the library:
+1. Add the `@egov` registry to your npm configuration (see the first step in the [general Installation instructions](https://github.com/e-gov/cvi#%EF%B8%8F-installation-instructions)).
+
+2. Install the library:
 ```
 npm i --save @egov/cvi-icons
 ```

--- a/libs/storybook/src/lib/stories/ng-diagrams.stories.mdx
+++ b/libs/storybook/src/lib/stories/ng-diagrams.stories.mdx
@@ -29,12 +29,14 @@ import diagramsPkg from '/libs/ng-diagrams/package.json'
 
 ## Installation
 
-1. Install the library
+1. Add the `@egov` registry to your npm configuration (see the first step in the [general Installation instructions](https://github.com/e-gov/cvi#%EF%B8%8F-installation-instructions)).
+
+2. Install the library
 ```
 npm i --save @egov/cvi-ng-diagrams
 ```
 
-2. Import `NgDiagramsModule` in your `app.module.ts`.
+3. Import `NgDiagramsModule` in your `app.module.ts`.
 
 ```ts
 

--- a/libs/storybook/src/lib/stories/styles/installation.stories.mdx
+++ b/libs/storybook/src/lib/stories/styles/installation.stories.mdx
@@ -19,11 +19,13 @@ import stylesPkg from '/libs/styles/package.json';
 
 The library styles are built using Sass (namely [Dart Sass](https://sass-lang.com/dart-sass)).
 
-1. Install the library
+1. Add the `@egov` registry to your npm configuration (see the first step in the [general Installation instructions](https://github.com/e-gov/cvi#%EF%B8%8F-installation-instructions)).
+
+2. Install the library
     ```
     npm i --save @egov/cvi-styles
     ```
-2. Include styles to a Sass file in the project:
+3. Include styles to a Sass file in the project:
     ```
     @use '@egov/cvi-styles/main';
     ```

